src/extension.ts

```diff
@@ -160,7 +160,7 @@ export async function attach(
                 DevToolsPanel.createOrShow(context, telemetryReporter, targetWebsocketUrl, runtimeConfig);
             } else if (useRetry) {
                 // Wait for a little bit until we retry
-                await new Promise((resolve, reject) => {
+                await new Promise<void>((resolve) => {

```

Comment:
why is this changing

Reply:
This isn't related to the change, but it is causing a linting warning on vscode's side.  Figured we should fix it.